src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { getSelectedTab } from '../../helper';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = getSelectedTab(tabs, selectedTabId);
+
+  const ARE_IDS_EQUAL = tab => tab.id === selectedTab.id;
```

Comment:
Bad function name. The function name should start with a verb and be written in `camelCase`